student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(_age, classroom, name = 'unknown', parent_permission: true)
+    super(nage, name, parent_permission: parent_permission)
```

Comment:
- [ OPTIONAL ] It looks like you made a typo here in the super parameters. You have `nage` instead of `age`. Kindly fix this 👍